ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge.cfg

```
@@ -108,6 +113,23 @@ router adaptive-virtual-topology
       avt profile PROD-AVT-POLICY-VIDEO id 4
       avt profile PROD-AVT-POLICY-MPLS-ONLY id 5
 !
+router internet-exit
+    !
+    exit-group ZSCALER-EXIT-POLICY-1_PRI
+        local connection ZSCALER-CONN-100
```

and adjust connection names
```suggestion
        local connection IE-Tunnel100
```